### PR TITLE
ArcSight py3

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5685

## Description
converted the integration of arcSight to py3 and added unittests

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py [hunk 1]
```diff
@@ -369,6 +370,15 @@ def get_query_viewer_results_command():
...
 
+#         {
+#             Event-Event ID:<Event-Event ID1>
+#             ID:<ID1>
+#         },
+#         {
+#             Event-Event ID:<Event-Event ID2>
+#             ID:<ID2>
+#         }
+
```
Comment (bziser) [inline Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py:381]: for what these comments?

File: Packs/ArcSightESM/ReleaseNotes/1_1_8.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ArcSight ESM v2
+
+- Documentation and metadata improvements.
```
Comment (bziser) [inline Packs/ArcSightESM/ReleaseNotes/1_1_8.md:6]: you can remove this

File: Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py [hunk 3]
```diff
@@ -872,50 +882,104 @@ def get_all_query_viewers_command():
...
+#
+#     elif demisto.command() == 'as-get-entries':
+#         get_entries_command()   # done
+#
+#     elif demisto.command() == 'as-add-entries':
+#         entries_command(func='addEntries')
+#
+#     elif demisto.command() == 'as-delete-entries':
+#         entries_command(func='deleteEntries')   # done
+#
```
Comment (bziser) [inline Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py:919]: you can delete these comments too, you moved it to the main

File: Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py [hunk 4]
```diff
@@ -872,50 +882,104 @@ def get_all_query_viewers_command():
...
+            test()
+            demisto.results('ok')
+
+        elif demisto.command() == 'as-fetch-incidents' or demisto.command() == 'fetch-incidents':
+            fetch()
 
-    elif demisto.command() == 'as-fetch-incidents' or demisto.command() == 'fetch-incidents':
-        fetch()
+        elif demisto.command() == 'as-get-matrix-data' or demisto.command() == 'as-get-query-viewer-results':
+            get_query_viewer_results_command()  # done
```
Comment (bziser) [inline Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2.py:944]: why needed the `#done` in all the commands?

**Review by bziser — COMMENTED** [review]
great

**Comment by merit-maita** [comment]
@bziser i will remove all the comments, i just needed to make sure the first built passed with the changes i made

**Review by bziser — APPROVED** [review]

**Pull request merged**